Sources/SwiftSentry/Options.swift

```diff
@@ -13,6 +13,7 @@ public final class Options {
   public var crashHandlerPath: URL?
   public var beforeSend: ((AnyObject) -> AnyObject)?
   public var debug: Bool = false
+  public var shutdownTimeout: Int?
```

Comment:
nit: might want to document the units here or consider using `TimeInterval` as the type https://developer.apple.com/documentation/foundation/timeinterval